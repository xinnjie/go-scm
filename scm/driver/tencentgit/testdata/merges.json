[
  {
    "labels": [
      "versions",
      "test_label"
    ],
    "id": 341756,
    "title": "add close-pr.md",
    "target_project_id": 179129,
    "target_branch": "master",
    "source_project_id": 179129,
    "source_branch": "branch-for-close-test",
    "state": "opened",
    "merge_status": "can_be_merged",
    "iid": 3,
    "description": "add close-pr.md",
    "created_at": "2021-08-24T11:28:58+0000",
    "updated_at": "2021-08-25T16:59:10+0000",
    "resolved_at": null,
    "merge_type": null,
    "assignee": null,
    "author": {
      "id": 176405,
      "username": "xinnjie",
      "web_url": "https://git.code.tencent.com/u/xinnjie",
      "name": "xinnjie",
      "state": "active",
      "avatar_url": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1."
    },
    "merge_commit_sha": null,
    "milestone": null,
    "necessary_reviewers": null,
    "suggestion_reviewers": null,
    "base_commit": "225163e0798965b54b6df9579e35f0725176a792",
    "target_commit": "225163e0798965b54b6df9579e35f0725176a792",
    "source_commit": "b230dc632847381015ad4beffbdcb4e4d23f775b",
    "project_id": 179129,
    "work_in_progress": false,
    "upvotes": 0,
    "downvotes": 0
  }
]