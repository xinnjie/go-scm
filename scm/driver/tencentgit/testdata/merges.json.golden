[
  {
    "Number": 3,
    "Title": "add close-pr.md",
    "Body": "add close-pr.md",
    "Labels": [
      {
        "ID": 0,
        "URL": "",
        "Name": "versions",
        "Description": "",
        "Color": ""
      },
      {
        "ID": 0,
        "URL": "",
        "Name": "test_label",
        "Description": "",
        "Color": ""
      }
    ],
    "Sha": "",
    "Ref": "refs/merge-requests/3/head",
    "Source": "branch-for-close-test",
    "Target": "master",
    "Base": {
      "Ref": "master",
      "Sha": "225163e0798965b54b6df9579e35f0725176a792",
      "Repo": {
        "ID": "179129",
        "Namespace": "xinnjie",
        "Name": "testme",
        "FullName": "xinnjie/testme",
        "Perm": {
          "Pull": true,
          "Push": false,
          "Admin": false
        },
        "Branch": "master",
        "Private": false,
        "Archived": false,
        "Clone": "https://git.code.tencent.com/xinnjie/testme.git",
        "CloneSSH": "git@git.code.tencent.com:xinnjie/testme.git",
        "Link": "https://git.code.tencent.com/xinnjie/testme",
        "Created": "2021-08-20T10:44:47Z",
        "Updated": "0001-01-01T00:00:00Z"
      }
    },
    "Head": {
      "Ref": "branch-for-close-test",
      "Sha": "b230dc632847381015ad4beffbdcb4e4d23f775b",
      "Repo": {
        "ID": "179129",
        "Namespace": "xinnjie",
        "Name": "testme",
        "FullName": "xinnjie/testme",
        "Perm": {
          "Pull": true,
          "Push": false,
          "Admin": false
        },
        "Branch": "master",
        "Private": false,
        "Archived": false,
        "Clone": "https://git.code.tencent.com/xinnjie/testme.git",
        "CloneSSH": "git@git.code.tencent.com:xinnjie/testme.git",
        "Link": "https://git.code.tencent.com/xinnjie/testme",
        "Created": "2021-08-20T10:44:47Z",
        "Updated": "0001-01-01T00:00:00Z"
      }
    },
    "Fork": "xinnjie/testme",
    "State": "open",
    "Closed": false,
    "Draft": false,
    "Merged": false,
    "Mergeable": true,
    "Rebaseable": false,
    "MergeableState": "mergeable",
    "MergeSha": "",
    "Author": {
      "ID": 176405,
      "Login": "xinnjie",
      "Name": "xinnjie",
      "Email": "",
      "Avatar": "https://git.code.tencent.com/uploads/user/avatar/176405/7e72ab97e19e430287c34c54af3bf7e1.",
      "Link": "https://git.code.tencent.com/u/xinnjie",
      "Created": "0001-01-01T00:00:00Z",
      "Updated": "0001-01-01T00:00:00Z"
    },
    "Assignees": null,
    "Reviewers": null,
    "Milestone": {
      "Number": 0,
      "ID": 0,
      "Title": "",
      "Description": "",
      "Link": "",
      "State": "",
      "DueDate": null
    },
    "Created": "2021-08-24T11:28:58Z",
    "Updated": "2021-08-25T16:59:10Z",
    "Link": "https://git.code.tencent.com/xinnjie/testme/merge_requests/3",
    "DiffLink": ""
  }
]
